packages/host/app/components/operator-mode.gts

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { Card } from 'https://cardstack.com/base/card-api';
+
+interface Signature {
+  Args: {
+    firstCardInStack: Card;
+  };
+}
+
+export default class OperatorMode extends Component<Signature> {
+  stack: Card[] = [];
+
+  constructor(owner: unknown, args: any) {
+    super(owner, args);
+    this.stack = [this.args.firstCardInStack];
+  }
+
+  <template>
+    <div class='operator-mode-desktop-overlay'>
+      <div class='operator-mode-card-stack'>
+        {{#each this.stack as |card|}}
+          <div class='operator-mode-stack-card'>
+            <card />
```

Comment:
not quite right, the `Card` is not actually a component. you need to get the component for the card based ont eh format you want to render (which is probably "isolated"). you can do that like this: https://github.com/cardstack/boxel-motion/blob/main/packages/host/app/components/preview.gts#L17-L19

Reply:
I’m that case the type is wrong, `this.stack` is not an array of `Card`. It’s an array of components. You should have gotten a type error somewhere since the type for this component’s args does not match the model.

Reply:
Also I would think that there should be a type error in the template since `card` is not a component.

Reply:
There was indeed a type error but I deferred solving that later - thanks for the tip! I used `ComponentLike` which clears the issue